compiler/passes/convert-uast.cpp

```diff
@@ -650,6 +665,23 @@ struct Converter {
             ? createBlockWithStmts(node->elseStmts())
             : nullptr;
 
+      Expr* cond = nullptr;
+
+      // TODO: Can 'ifVars' happen in expression-level conditionals?
+      if (auto ifVar = node->condition()->toVariable()) {
+        assert(ifVar->kind() == uast::Variable::CONST ||
+               ifVar->kind() == uast::Variable::VAR);
+        assert(ifVar->initExpression());
+        auto identDef = ifVar->name().c_str();
```

Comment:
This is the name of it, not a DefExpr, so suggest renaming to e.g. varName.

Reply:
Should I also change the arg names of the old parser helpers? That's what I've been following, but perhaps not the most sound reasoning.

Reply:
I think it'd be reasonable to do so but it's not a requirement